src/e3/testsuite/__init__.py

```diff
@@ -1062,9 +1063,14 @@ def collect_result(job: Job) -> bool:
             return False
 
         # Create a scheduler to run all fragments for the testsuite main loop
+
+        jobs = self.main.args.jobs
+        if not self.main.args.jobs > 0:
+            jobs = multiprocessing.cpu_count()
```

Comment:
According to https://docs.python.org/3/library/multiprocessing.html#multiprocessing.cpu_count, this may raise a `NotImplementedError` exception: it should be handled here. Alternatively, using https://docs.python.org/3/library/os.html#os.cpu_count could be simpler: `os.cpu_count() or 1`